purchases/src/main/kotlin/com/revenuecat/purchases/common/OfferingParser.kt

```diff
@@ -52,6 +54,13 @@ internal abstract class OfferingParser {
             }
         }
 
+        val targeting: Targeting? = offeringsJson.optJSONObject("targeting")?.let {
+            val revision = it.getInt("revision")
+            val ruleId = it.getString("rule_id")
```

Comment:
We could write some tests in the `OfferingsFactoryTest` to actually perform the parsing of these I think